packages/comet-uswds/src/components/date-picker/date-picker.tsx

```diff
@@ -54,7 +54,11 @@ export const DatePicker = ({
 
   useLayoutEffect(() => {
     const datePickerElement = datePickerRef.current as HTMLInputElement;
-    datePicker.on(datePickerElement);
+    const datePickerWrapper = datePickerElement.querySelector(
+      '.usa-date-picker__wrapper',
+    ) as HTMLInputElement;
```

Comment:
should this be HTMLElement vs HTMLInputElement?

Reply:
Fixed